playbooks/roles/cluster/defaults/main.yml

```diff
@@ -1,5 +1,82 @@
 ---
+# common
 cluster_credential_git_user: ''
 cluster_credential_git_user_group: ''
 cluster_credential_git_repo: ''
 cluster_credential_git_ref: ''
+cluster_project_bootstrap_scm_type: git
+cluster_project_bootstrap_cluster_scm_clean: true
+cluster_project_bootstrap_cluster_scm_update_on_launch: true
+cluster_project_bootstrap_cluster_scm_delete_on_update: true
+
+#                         #
+# Cluster create projects #
+#                         #
+
+# openshift-ansible-ops
+cluster_project_oao_bootstrap_scm_branch: tower
+cluster_project_oao_bootstrap_scm_url: "git@github.com:StevenTobin/openshift-ansible-ops.git"
```

Comment:
These repos require ssh keys to access? Should these be blank or pointing at a public repo by default, and then overridden in the private credential repos ?

Reply:
Well there's no point in pointing at anything else and there is no public version.

I could blank them out I suppose.

Reply:
Yeah id say blank em out by default, and then have matching group_vars in the prod/dev credential repos.

but how will this work in the long run? do we hope to make those repos or forks of them public?

Reply:
Yeah I want to move the functionality into the configuration repo eventually